CHANGELOG.asciidoc

```diff
@@ -3,6 +3,158 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.1.0]]
+=== Beats version 8.1.0
+https://github.com/elastic/beats/compare/v8.0.1...v8.1.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+
+*Auditbeat*
+
+
+*Filebeat*
+
+- Remove Recorded Future integration from threatintel module. {pull}30564[30564]
```

Comment:
@adriansr I stumbled over this text as it talks about the integration but my understanding is it only removes a dataset?

Reply:
@ruflin yes I meant Filebeat's dataset.

Reply:
I see @andresrc already corrected it. Thanks